Update category-list.md (#8313)
magento/devdocs: src/guides/v2.3/graphql/queries/category-list.md

}
```

### Return the categoryList by url_key filters

> The following query returns information about category by url_key filters. You have to pass url_key value without suffix with `eq` or `in` keyword. Get the categoryList response from the `Gear` category.

Reviewer: ```suggestion
The following query returns information about categories by `url_key` filters. You have to pass the `url_key` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear` category.
```

> The following query returns information about categories by `url_key` filters. You have to pass the `url_key` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear` category.

Reviewer 2: ```suggestion
The following query returns information about the Gear category using a `url_key` filter. You must pass the `url_key` value without a suffix and specify either the `eq` or `in` keyword.
```

**Request:**

```graphql
{
  categoryList(filters: { url_key: { eq: "gear" } }) {
    id
    level
    name
    path
    url_path
    url_key
    children_count
  }
}
```

**Response:**

```json
{
  "data": {
    "categoryList": [
      {
        "id": 3,
        "level": 2,
        "name": "Gear",
        "path": "1/2/3",
        "url_path": "gear",
        "url_key": "gear",
        "children_count": "3"
      }
    ]
  }
}
```

### Return the categoryList by url_path filters

> The following query returns information about category by url_path filters. You have to pass url_path value without suffix with `eq` or `in` keyword. Get the categoryList response from the `Gear -> Bags` category.

Reviewer: ```suggestion
The following query returns information about categories by `url_path` filters. You have to pass the `url_path` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear -> Bags` category.
```

> The following query returns information about categories by `url_path` filters. You have to pass the `url_path` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear -> Bags` category.

Reviewer 2: ```suggestion
The following query returns information about the Gear > Bags` category using the `url_path` filter. Do not specify a suffix in the `url_path` value. The `url_path` filter accepts either the `eq` or `in` keyword.
```
